test/provisioner/provision_ibmcloud_initializer.go

```diff
@@ -81,14 +81,13 @@ func initLogger() {
 
 func initProperties(properties map[string]string) error {
 	IBMCloudProps = &IBMCloudProperties{
-		ApiKey:        properties["APIKEY"],
-		Bucket:        properties["COS_BUCKET"],
-		ClusterName:   properties["CLUSTER_NAME"],
-		CosApiKey:     properties["COS_APIKEY"],
-		CosInstanceID: properties["COS_INSTANCE_ID"],
-		CosServiceURL: properties["COS_SERVICE_URL"],
-		IamServiceURL: properties["IAM_SERVICE_URL"],
-		// IsSelfManaged    : properties["IS_SELF_MANAGED_CLUSTER"]
+		ApiKey:          properties["APIKEY"],
+		Bucket:          properties["COS_BUCKET"],
+		ClusterName:     properties["CLUSTER_NAME"],
+		CosApiKey:       properties["COS_APIKEY"],
+		CosInstanceID:   properties["COS_INSTANCE_ID"],
+		CosServiceURL:   properties["COS_SERVICE_URL"],
+		IamServiceURL:   properties["IAM_SERVICE_URL"],
 		InstanceProfile: properties["INSTANCE_PROFILE_NAME"],
 		PodvmImageID:    properties["PODVM_IMAGE_ID"],
```

Comment:
I think that `PODVM_IMAGE_ID` has been removed from the default `provision_ibmcloud.properties` files, so maybe we need to add it back. I appreciate that longer term we might want to string `./cluster-provisioner -action=uploadimage` into provision and be able to re-use the output, but I don't think we are quite ready for that yet?